packages/accounts-discord/package.js

```
@@ -0,0 +1,22 @@
+Package.onUse(api => {
+  api.use('ecmascript');
+  api.use('accounts-base', ['client', 'server']);
+  api.imply('accounts-base', ['client', 'server']);
+
+  api.use('accounts-oauth', ['client', 'server']);
+  api.use('discord-oauth');
+  api.imply('discord-oauth');
+
+  api.use(
+    ['accounts-ui', 'discord-config-ui'],
+    ['client', 'server'],
```

ESLint

[error] Replace `⏎····['accounts-ui',·'discord-config-ui'],⏎····['client',·'server'],` with `['accounts-ui',·'discord-config-ui'],·['client',·'server'],·{`

(prettier/prettier)